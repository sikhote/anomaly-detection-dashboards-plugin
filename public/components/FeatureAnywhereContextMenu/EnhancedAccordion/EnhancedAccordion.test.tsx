/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */

import React from 'react';
import { shallow } from 'enzyme';
import EnhancedAccordion from './EnhancedAccordion';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe('EnhancedAccordion', () => {
  test('renders', () => {
    const wrapper = shallow(<EnhancedAccordion />);
    expect(wrapper).toMatchSnapshot();
  });
});