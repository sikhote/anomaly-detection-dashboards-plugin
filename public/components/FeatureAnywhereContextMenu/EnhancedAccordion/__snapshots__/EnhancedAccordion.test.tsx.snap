// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EnhancedAccordion renders 1`] = `ShallowWrapper {}`;
